lib/Conversion/ImportVerilog/Statements.cpp

```diff
@@ -578,6 +580,83 @@ struct StmtVisitor {
       return true;
     }
 
+    // Display and Write Tasks (`$display[boh]?` or `$write[boh]?`)
+
+    // Check for a `$display` or `$write` prefix.
+    bool isDisplay = false;     // display or write
+    bool appendNewline = false; // display
+    StringRef name = subroutine.name;
```

Comment:
nit: this `name` variable looks like generic but it cannot be used other than handling `display/write` due to mutation. To avoid confusion can we change the variable name or create a local scope for handling `display/write`?

Reply:
Good point. It might be worthwhile factoring the different flavors of builtins out into separate functions, and calling them in sequence to try and match the subroutine :thinking:. I'll just rename the variable for now.